ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
```

Comment:
You're absolutely right! I can confirm a proper installed universal will return exit code 0 from `node tests/all-tests.js` on node.js 0.10.36, 0.10.40 and 4.2.2 equally.